packages/opentelemetry-instrumentation-langchain/tests/test_tool_call_content.py

```
@@ -49,46 +49,46 @@ def test_assistant_message_with_tool_calls_includes_content():
     call_args = [call[0] for call in mock_span.set_attribute.call_args_list]
     attributes = {args[0]: args[1] for args in call_args}
 
-    assert f"{SpanAttributes.LLM_PROMPTS}.0.role" in attributes
-    assert attributes[f"{SpanAttributes.LLM_PROMPTS}.0.role"] == "user"
-    assert f"{SpanAttributes.LLM_PROMPTS}.0.content" in attributes
+    assert f"{GenAIAttributes.GEN_AI_PROMPT}.0.role" in attributes
```

Ruff (0.12.2)

Undefined name `GenAIAttributes`

(F821)